functions/index.js

```diff
@@ -15,6 +21,21 @@ exports.helloWorld = functions.https.onRequest((request, response) => {
 	 }
 });
 
+exports.pullDataFromLocalCSVFile = functions.https.onRequest((request, response) => {
```

Comment:
I don't see where this function is used.  If it is something you're using for testing purposes (and not intended to be part of the working system), would you clarify that in a comment at the beginning of the function?  Thanks 🙏

Reply:
Yes. This is to be able to test with a file saved locally. I will add a comment